implementations/index.md

```diff
@@ -212,9 +212,11 @@ the moment.
   * [JSONAPI::Utils](https://github.com/b2beauty/jsonapi-utils) works on top of [JSONAPI::Resources](https://github.com/cerebris/jsonapi-resources) taking advantage of its resource-driven style and bringing a Rails way to build modern APIs with no or less learning curve.
   * [Caprese](https://github.com/nicklandgrebe/caprese) An opinionated Rails library for creating JSON:API servers that lets you focus on customizing the behavior of your endpoints rather than the dirty work of setting them up. Leverages the power of [ActiveModel::Serializer](https://github.com/rails-api/active_model_serializers).
   * [JSONAPI Suite](https://jsonapi-suite.github.io/jsonapi_suite)
-  facilitates a server capable of deep querying and nested writes. Works
-  with any ORM or datastore; comes with integration test helpers and
-  automatic swagger documentation.
```

Comment:
You accidentally removed some description from the other implementation, could you add that back?

Reply:
Apologies @lode, I think I rushed while copy/pasting... :see_no_evil: 
Fixed now.